### DISCUSSION TITLE
I can't find the new X (formerly twitter) logo.

### DISCUSSION THREAD

**Discussion by Adityapandey1302** [lucide-icons/lucide#2689] [Help] [answered]
I have searched everywhere for the X (formerly twitter) icon import from Lucide-react but can't find it anywhere. I guess I'm not finding it or the logo is not present. Kindly guide.

**Comment by karsa-mistmere** [comment, marked as answer]
Brand icons are deprecated, see #670 

The brand logo for X won't ever be added, Twitter will eventually be removed.
